crates/astria-core/src/primitive/v1/mod.rs

```diff
@@ -488,6 +490,104 @@ where
     tree
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
```

Comment:
The transaction ID should be `Copy` (which it can be if you change the internal type to `[u8; 32]`).